### ISSUE TITLE
Completely update godot-vrm addon to latest version

### ISSUE THREAD

**Issue by you-win** [you-win/openseeface-gd#20]
The majority of the godot-vrm addon code is actually the code from ~January 2021 and there have been significant updates since then. The application is actually currently able to run off of Godot's 3.x branch but loading `.vrm` files is broken because of the outdated addon.

After chatting with the V-Sekai team, it looks like my modifications to the addon were unnecessary, so it should be possible to use the godot-vrm addon as a git submodule.

- [ ] update godot-vrm to be a git submodule
- [x] update `VRMMappings.gd` to pull bone mappings from vrm_meta.gd instead of intercepting the mappings inside of `import_vrm.gd`. `VRMMappings.gd` is used for mapping facial blend shapes (e.g. blinking, mouth open, etc) to facial tracking

**Comment by you-win** [comment]
still waiting on merge from upstream godot-vrm repo

**Comment by fire** [comment]
What things are requested?

**Comment by you-win** [comment]
In order to use godot-vrm as a git submodule, the [changes from this pr](https://github.com/V-Sekai/godot-vrm/pull/20#pullrequestreview-719666866) are required.

**Comment by fire** [comment]
Let's update this issue.

**Comment by you-win** [comment]
The issue is still technically blocked since the pr hasn't been merged yet + I have several other changes on my local version to work around `Resource` files losing data after being created (i.e. `Resource` files from godot-vrm have been changed to `Reference` and have the `duplicate()` function manually implemented.

If the pr will never be merged, then both the pr and this issue can be closed

**Comment by you-win** [comment]
Closing this issue for now, my fork of godot-vrm has diverged fairly heavily. Might open a new issue if a Godot 4 port of openseeface-gd comes around.
